examples/send_one.py

```diff
@@ -1,20 +1,31 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+This example shows how sending a single message works.
+"""
+
 from __future__ import print_function
-import can
 
+import can
 
 def send_one():
     bus = can.interface.Bus(bustype='pcan', channel='PCAN_USBBUS1', bitrate=250000)
```

Comment:
There should also be a `bus = can.interface.Bus()` version with a comment refering to env/file config

Reply:
Yup. See the last commit below.